developer/wip/for-contributors/contributor-handbook.md

```diff
@@ -580,7 +583,80 @@ The sample expresses that a set of assets identified by their ID be made availab
 
 > Note that asset selector expressions are always logically conjoined using an "AND" operation.
 
-#### 2.1.4 Contract agreements
+#### 2.1.4 Contract negotiations
+
+If a connector fulfills the [contract policy](#213-contract-definitions), it may initiate the negotiation of a contract for
+a particular asset. During that negotiation, both parties can send offers and counter-offers that can contain altered
+terms (= policy) as any human would in a negotiation, and the counter-party may accept or reject them.
+
+Contract negotiations have a few key aspects:
+
+- they target _one_ asset
+- they take place between a _provider_ and a _consumer_ connector
+- they cannot be changed by the user directly
+- users can only decline, terminate or cancel them
+
+As a side note it is also important to note that contract offers are _ephemeral_ objects as they are generated
+on-the-fly for a particular participant, and they are never persisted in a database and thus cannot be queried through
+any API.
+
+Contract negotiations are asynchronous in nature. That means after initiating them, they become (potentially
+long-running) stateful processes that are advanced by an internal [state machine](#221-state-machines).
+The current state of the negotiation can be queried and altered through the management API.
+
+> In the current iteration all contract offers are automatically accepted by both parties if the contract
```

Comment:
we already have them -> guards: https://github.com/eclipse-edc/Connector/blob/de80081d547c66f127dd32e1a8b27d9fca2d4410/docs/developer/state-machine.md#guards